Mathmagician/Fibonacci.cs

```diff
@@ -4,9 +4,51 @@
 using System.Text;
 using System.Threading.Tasks;
 
+
 namespace Mathmagician
 {
     public class Fibonacci
     {
+        
+        public static int IsFibonacci(int n)
+        {
+            int a = 0;
+            int b = 1;
+            // In N steps compute Fibonacci sequence iteratively.
+            for (int i = 0; i < n; i++)
```

Comment:
@guthb 
Performance could be improved by doing just the Fibonacci equation in this method and then calling it in your forLoop in the ReturnFibonacciSequence. Sorry I didn't catch that tonight!